collections/hrsi-rlie-s1.yaml

```diff
@@ -0,0 +1,140 @@
+AdditionalInfoExternal:
+  Title: Additional info
+  Path: hrsi-rlie-s1/README.MD
+EDCBrowser: ""
+EOBrowser: ""
+Image: hrsi-rlie-s1/thumbnail.png
+Resolution: 20m
+GeographicalCoverage: Europe  (Lat; 26N 72N Lon; -25W 45E)
+TemporalAvailability: September 1, 2016 - ongoing
+UpdateFrequency: NRT, between 6 and 12 hours after the data sensing time.
+BandInformation:
+  Table:
+    Columns:
+      - Name: Name
+        Title: Name
+      - Name: Description
+        Title: Description
+      - Name: Units
+        Title: Units
+      - Name: Range
+        Title: Range
+      - Name: Sentinel Hub Units
+        Title: Sentinel Hub Units
+      - Name: Digital Range
+        Title: Digital Range
+      - Name: No data
+        Title: No data
+    Rows:
+      - Name: RLIE
+        Description: River and lake ice extent
+        Units: Not applicable
+        Range: Not applicable
+        Sentinel Hub Units: DN
+        Digital Range: 1, 100
+        No data: 255
+      - Name: QC
+        Description: Quality layer providing basic assessment of RLIE quality
+        Units: Not applicable
+        Range: Not applicable
+        Sentinel Hub Units: DN
+        Digital Range: 0 - 3
+        No data: 255
+      - Name: QCFLAGS
+        Description: Quality flags
+        Units: bit; Additional info <a href="https://land.copernicus.eu/user-corner/technical-library/hrsi-ice-s1-atbd/">here</a>.
+        Range: 1 - 7
+        Sentinel Hub Units: Not applicable
+        Digital Range: Not applicable
+        No data: Not applicable
+Contact: "[Copernicus Land Monitoring Service Contact](https://land.copernicus.eu/contact)"
+Provider: "[European Environment Agency](https://www.eea.europa.eu/)"
+ManagedBy: "[Sentinel Hub](https://www.sentinel-hub.com/)"
+Tags:
+  - hrsi
+  - water
+  - river
+  - lake
+  - ice
+  - CLMS
+  - derived data
+  - open data
+License: Freely and openly accessible according to the Copernicus data and information policy. ([Read more](https://land.copernicus.eu/terms-of-use)).
+LicenseType: proprietary
+LicenseUrl: https://land.copernicus.eu/terms-of-use
+Resources:
+  - Group: Sentinel Hub Resources
+    EndPoint: https://creodias.sentinel-hub.com
+    Name: Sentinel Hub
+    Role: processor
+    Type: byoc-fe2c0caf-fbe8-49a1-91ae-e730ea65116a
+    CollectionId: fe2c0caf-fbe8-49a1-91ae-e730ea65116a
+    Primary: true
+  - Group: xcube Resources
+    DatasetName: CUSTOM
+    CollectionId: fe2c0caf-fbe8-49a1-91ae-e730ea65116a
+CustomScripts:
+  Title: Collection of Copernicus services data custom scripts
+  URL: https://custom-scripts.sentinel-hub.com/#copernicus_services
+Configurations:
+  - href: "https://docs.sentinel-hub.com/api/latest/api/process/"
+    rel: "about"
+    type: "text/html"
+    title: "Details about running Evalscripts"
+  - href: "https://custom-scripts.sentinel-hub.com/copernicus_services/hrsi-rlie-s1/"
```

Comment:
Needs a direct link to the evalscript js

I think there is a issue with the customs script repo- it doesnt give a direct link. created an issue to correct it

Here is the right link 

https://custom-scripts.sentinel-hub.com/copernicus_services/hrsi/hrsi-rlie-s1/script.js